api/src/test/java/org/openmrs/api/db/hibernate/HibernateOrderDAOTest.java

```diff
@@ -173,4 +177,92 @@ public void deleteOrderGroupAttributeType_shouldDeleteOrderGroupAttributeTypeFro
 		dao.deleteOrderGroupAttributeType(orderGroupAttributeType);
 		assertNull(dao.getOrderGroupAttributeByUuid(UUID));
 	}
+
+	/**
+	 * @see {@link HibernateOrderDAO#getAllOrderAttributeTypes()}
+	 * @throws Exception
+	 */
+	@Test
+	public void getAllOrderAttributeTypes_shouldGetAllOrderAttributeTypes() {
+		List<OrderAttributeType> orderAttributeTypes = dao.getAllOrderAttributeTypes();
```

Comment:
You may not need this .. could verily do it as

`assertEquals(dao.getAllOrderAttributeTypes(), 4);`